make/modules/java.base/Lib.gmk

```diff
@@ -157,8 +157,14 @@ endif
 
 ################################################################################
 ## Build libsyslookup
+## The BSDLIBC depenency on OpenBSD is needed to access libc symbols. On OpenBSD
```

Comment:
Nit: depenency -> dependency

Reply:
Thank you for looking it over. My spelling and grammar has always been my weakest skill.